README.md

```diff
@@ -23,21 +23,73 @@
+work with your favorite Git forges, too.
+
+Jujutsu combines many distinct design features from other version control
+systems into a single user interface. Those other systems include:
+
+- **Git**: Like Git, we make an effort to [be fast][perf], from a snappy
+  UX to efficient algorithms. It also by default shares Git's data model, for
+  wide interoperability and ease of onboarding.
+
+- **Mercurial & Sapling**: Many Mercurial inspired features include the [revset]
+  language to select commits. There is [no explicit index][no-index] or "staging
+  area". Branches are "anonymous" like Mercurial, so you don't need to make up
+  a name for each small change. Primitives for rewriting history are powerful
+  and simple.
+
+- **Pijul & Darcs**: Jujutsu keeps track of conflicts as [first-class objects][conflicts]
+  in its model; it tracks them as if it tracked any other file. While not as
+  rigorous as patch-based systems like Pijul and Darcs, the effect is that many
+  forms of conflict resolution can be performed and propagated automatically.
+
+[perf]: https://github.com/martinvonz/jj/discussions/49
+[revset]: https://martinvonz.github.io/jj/latest/revsets/
+[no-index]: https://martinvonz.github.io/jj/latest/git-comparison/#the-index
+[conflicts]: https://martinvonz.github.io/jj/latest/conflicts/
+
+And it adds several innovative features of its own that those don't have:
+
+- **Working-copy-as-a-commit**: Changes to files are [recorded automatically][wcc]
+  as normal commits, and amended on every subsequent change. This "snapshot"
+  design simplifies the user-facing data model, simplifies internal algorithms,
+  and completely subsumes features like the Git's stash and index/staging-area.
+
+- **Operation log, undo**: Jujutsu records every operation that is performed on the
+  repository, from commits, to pulls, to pushes. This makes debugging problems like
+  "what just happened?" or "how did I end up here?" easier, *especially* when you're
+  helping your coworker answer those questions! And because everything is recorded,
+  you can undo that mistake you just made with ease. Finally: version control
+  has entered [the 1960s][undo-history]!
+
+- **Safe, concurrent replication**: Have you ever wanted to store your version
+  controlled repositories inside a Dropbox folder? No? Well now you can!
+  Fundamentally the problem with using systems like Dropbox for Git or
+  Mercurial, or backing them up with `rsync`, is that that they rely on *local
+  filesystem operations* being atomic, serialized, and non-concurrent with
+  respect to other writes—which is _not_ true when operating on distributed
+  file systems or when operations (like copying files to S3) happen while
+  lock files are being held. Jujutsu is instead designed to be [safe under
+  concurrent][conc-safety] scenarios; rsync and dropbox will never *corrupt*
+  your repository. The worst that will happen is that it will expose conflicts
```

Comment:
There has been a few bugs in this area, however, especially related to the Git backend. Are @ilyagr and @yuja comfortable with this wording?

Reply:
Yes, please let me know. Actually, I think if we do believe this is stable enough to advertise, we should strongly write tests and examples for it that are "user understandable" e.g. how to use rsync for backup and what could happen if something goes wrong. That would mean people can effectively just use dumb blob storage as repository backup, which is pretty cool!

Reply:
I also wonder what @yuja thinks.

I'm not very comfortable advertising this to users at the moment except as an ambition. For example, you could say:

> Well, now you can try it! (Note: jj has some bugs and is not well-tested in this area; making this entirely safe is more of an ambition than reality at the moment).

And towards the end:

> ...  rsync and dropbox should never *corrupt* a `jj` repository. The worst that should happen...

Reply:
> I'm not very comfortable advertising this to users at the moment except as an ambition.

Agreed. While the GitBackend issue wouldn't lead to data loss, the whole rsync story isn't battle-tested and there may be unknown bugs.

> the problem with using systems like Dropbox for Git or
> Mercurial, or backing them up with `rsync`, is that that they rely on *local
> filesystem operations* being atomic, serialized, and non-concurrent

It's true that Mercurial data model isn't rsync-safe, but I think Git repo is mostly okay. afaict, JJ is slightly safer than Git since refs (and heads) can be updated concurrently.

Reply:
I've changed the wording, and also put this under an 'experimental' section along with workspaces. I've added automatic rebase in its stead to make the bullet points look a bit beefer. Let me know what you think!